docs/src/merge-report-js.md

```diff
@@ -0,0 +1,41 @@
+---
+id: mock
```

Comment:
```suggestion
id: merge-report
```

Reply:
This should be test-merge-report instead?

Reply:
Done. Renamed to test-merge-report-js.md